packages/vscode-extension/src/envTree.ts

```diff
@@ -255,11 +255,23 @@ async function appendSubscriptionAndResourceGroupNode(env: string): Promise<void
   ) {
     const envSubItems: TreeItem[] = [];
     const subscriptionInfo = await getSubscriptionInfoFromEnv(env);
+
     if (subscriptionInfo) {
       const subscriptionTreeItem: TreeItem = {
         commandId: `fx-extension.environment.subscription.${env}`,
         contextValue: "openSubscriptionInPortal",
         label: subscriptionInfo.subscriptionName ?? subscriptionInfo.subscriptionId,
+        description: subscriptionInfo.subscriptionId,
+        tooltip: {
+          isMarkdown: false,
+          value: `'${env}' environment is provisioned in Azure subscription '${
```

Comment:
Can you extract the string to Strings.json for later localization?

Reply:
Sure. Thanks for suggestion.